G_channels.md

```diff
@@ -92,10 +89,10 @@ The first priority of your channels is to authorize clients to join a given topi
 defmodule HelloPhoenix.RoomChannel do
   use Phoenix.Channel
 
-  def join("rooms:lobby", auth_msg, socket) do
+  def join("rooms:lobby", _message, socket) do
     {:ok, socket}
   end
-  def join("rooms:" <> _private_room_id, _auth_msg, socket) do
```

Comment:
the subtopic is the private room id, so lets' make this line `def join("rooms:" <> _private_room_id, _params, socket) do`